docs/semgrep-app/findings.md

```diff
@@ -97,7 +97,7 @@ The Findings page consists of:
 
 **Triaging** means prioritizing a finding based on a policy or criteria set by your team or organization. While severity is a factor in triage, your organization may define additional criteria based on coding standards, business, or product goals.
 
-Semgrep App assists in the triage process through the use of **comments**, **triage**, and automatically marks findings as fixed when they are no longer found.
+Semgrep App assists in the status tracking process through the use of **triage** and **bulk triage**, and automatically marks findings as fixed when they are no longer found. It also removes findings when a user disables or a removes a rule, closes a PR without merging it, or stops scanning a particular file.
```

Comment:
With regards to Findings page, we probably do not have to inform about: `closes a PR without merging it, or stops scanning a particular file.` Can we remove the last part of the sentence?